libavcodec/jpeg2000.h

```diff
@@ -192,6 +200,8 @@ typedef struct Jpeg2000Cblk {
     /* specific to HT code-blocks */
     int zbp;
     int pass_lengths[2];
+    uint8_t modes; // copy of SPcod/SPcoc field to parse HT-MIXED mode
+    uint8_t ht_plhd; // are we looking for HT placeholder passes?
```

Comment:
Shouldn't this be an enum, just like `JPEG2000_Ccap15_b14_15_params`?

Reply:
`modes` are already defined as MACROs in the original code. `ht_plhd` can be ENUM (preferable.)